src/lepton/jpgcoder.cc

```diff
@@ -265,6 +265,7 @@ bool write_ujpg(std::vector<ThreadHandoff> row_thread_handoffs,
 bool read_ujpg( void );
 unsigned char read_fixed_ujpg_header( void );
 bool reset_buffers( void );
+void prep_for_new_file( void );
```

Comment:
This commit also includes bugfix at https://github.com/interiorem/lepton/commit/438536018fdabe5d81f1cdcaac1e0abfac102f5c#diff-a4bc741b423893f19722580a52f91b37L4427

It may be excluded since later I further fixed these lines.

Overall this commit streamlined freeing of these vars, in particular moving some corresponding code from prep_for_new_file() to reset_buffers(), plus moved prep_for_new_file() to the same place as reset_buffers(), plus removed a few unused code lines, plus abovementioned bugfix.

Except for bugfix, rest of changes don't worth mentioning in the commit msg.

Reply:
I mean I don't understand why the variables at the top of commit and `prep_for_new_file()` were shuffled.